### PR TITLE
Upgrade Typescript etc.

### PR DESCRIPTION
**Pull request by Jskobos**
Package updates:

- Typescript (3.7.3)
- @types/ramda (0.26.33)
- prettier (1.19.1)
- tslint (5.20.1)

## Notes

Please test that everything is fine, console output is as before, Prettier works in your editor/on commit, etc.

### CODE REVIEW

**Comment by jstamis-linode** [comment]
I was able to do a yarn clean and yarn up to get things running.  The Jest tests passed however a console error appears on the Longview Graphs tests. Since longview is new code should we fix this now?
``` Console

    console.error ../../node_modules/react-dom/cjs/react-dom.development.js:530
      Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

      Check the render method of `LineGraph`.
          in Line (created by LineGraph)
          in div (created by LineGraph)
          in div (created by LineGraph)
          in LineGraph (created by LongviewLineGraph)
          in div (created by LongviewLineGraph)
          in LongviewLineGraph (created by Graphs)
          in div (created by Graphs)
          in div (created by Graphs)
          in Graphs (created by WithTheme(Graphs))
          in WithTheme(Graphs)
          in WithTheme(Graphs)
          in Router (created by MemoryRouter)
          in MemoryRouter
          in ThemeProvider (created by PreferenceToggle)
          in PreferenceToggle
          in PreferenceToggle (created by Connect(Component))
          in Connect(Component) (created by PreferenceToggle)
          in PreferenceToggle
          in PreferenceToggle (created by Connect(Component))
          in Connect(Component) (created by LinodeThemeWrapper)
          in LinodeThemeWrapper (created by Connect(LinodeThemeWrapper))
          in Connect(LinodeThemeWrapper)
          in Provider```

A note to our versioning. If minor updates can actually be breaking changes (React for instance) we should use the `~` instead of the `^` to make sure we keep our dependencies in line with the minor minor version.

**Comment by Jskobos** [comment]
@jstamis-linode Fixed the `~`s. We should fix the ref warning, but it's already in develop (I think) so I'd like to handle it in a separate PR.

**Comment by jstamis-linode** [comment]
@Jskobos Ok that sounds.

File: packages/manager/src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx [hunk 1]
```diff
@@ -155,7 +153,7 @@ interface StateProps {
 }
 
 const mapStateToProps: MapState<StateProps, Props> = state => ({
-  username: path(['data', 'username'], state.__resources.profile)
+  username: state.__resources?.profile?.data?.username
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx]: 🎉

**Pull request merged**